From 908fa9fee99913a4b728bfda0e6fe50a66546446 Mon Sep 17 00:00:00 2001
From: Jun Li <r65092@freescale.com>
Date: Fri, 5 Dec 2008 16:28:29 +0800
Subject: [PATCH] ENGR00101517 Fix utmi build error without pmic mc13892 defined in i.MX35.

arch/arm/plat-mxc/utmixc.c use mc13892 for i.MX51, but it's not controlled
by mc13892 macro, cause i.MX35 build error.

Signed-off-by: Li Jun <r65092@freescale.com>
---
 arch/arm/plat-mxc/utmixc.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/utmixc.c b/arch/arm/plat-mxc/utmixc.c
index 4a46b54..d6e8533 100644
--- a/arch/arm/plat-mxc/utmixc.c
+++ b/arch/arm/plat-mxc/utmixc.c
@@ -59,6 +59,7 @@ static void set_power(struct fsl_xcvr_ops *this,
 			regulator_disable(usbotg_regux);
 		}
 		regulator_put(usbotg_regux, dev);
+#if defined(CONFIG_MXC_PMIC_MC13892_MODULE) || defined(CONFIG_MXC_PMIC_MC13892)
 	} else if (machine_is_mx51_3ds()) {
 		unsigned int value;
 
@@ -77,6 +78,7 @@ static void set_power(struct fsl_xcvr_ops *this,
 		else
 			regulator_disable(usbotg_regux);
 		regulator_put(usbotg_regux, dev);
+#endif
 	}
 }
 
-- 
1.5.4.4

